apps/demo-app/package.json

```diff
@@ -1,17 +1,22 @@
 {
-  "name": "demo-app",
+  "name": "@sit-onyx/demo-app",
+  "version": "0.0.1",
   "private": true,
-  "version": "0.0.0",
-  "type": "module",
-  "author": "Schwarz IT KG",
-  "license": "Apache-2.0",
-  "engines": {
-    "node": ">=20"
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/SchwarzIT/onyx",
+    "directory": "apps/demo-app"
   },
+  "license": "Apache-2.0",
+  "author": "Schwarz IT KG",
+  "type": "module",
+  "files": [
```

Comment:
Since we are not publishing the demo app we can remove the files, right?
This declaration wouldn't work anyways because Nuxt bundles the required node_modules in the output folder but since they are in the .gitignore, they will not be included in the published package

Same for the other apps :)